Dockerfile

```diff
@@ -4,11 +4,11 @@ MAINTAINER Alexander Ilyin <alexander@ilyin.eu>
 
 # TEAMCITY INTERNAL ENVIRONMENT VARIABLES
 ENV TEAMCITY_DATA_PATH /root/.BuildServer
-ENV TEAMCITY_SERVER_MEM_OPTS -Xmx750m -XX:MaxPermSize=270m
-ENV TEAMCITY_SERVER_OPTS -Dteamcity.git.fetch.separate.process=false
+ENV TEAMCITY_SERVER_MEM_OPTS -Xmx750m
+ENV TEAMCITY_SERVER_OPTS -Dteamcity.git.fetch.separate.process=false -Dteamcity.agent.communicationProtocols=polling
```

Comment:
@nafg I have not noticed this option. Why do you need it? What was the use case?